spec/rubocop/cop/rails/where_missing_spec.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::WhereMissing, :config do
+  context 'Rails 6.1', :rails61 do
+    it 'registers an offense when using `left_joins(foo).where(foos: {id: nil})`' do
+      expect_offense(<<~RUBY)
+        Foo.left_joins(:foo).where(foos: { id: nil }).where(bar: "bar")
+            ^^^^^^^^^^^^^^^^ Use `where.missing(:x)` instead of `left_joins(:x).where(xs: { id: nil })`.
```

Comment:
It looks like it can show the actual value.

```suggestion
            ^^^^^^^^^^^^^^^^ Use `where.missing(:foo)` instead of `left_joins(:foo).where(foos: { id: nil })`.
```